server/service/service_osquery.go

```diff
@@ -1147,14 +1149,14 @@ func (svc *Service) SubmitDistributedQueryResults(ctx context.Context, results f
 			// osquery docs say any nonzero (string) value for
 			// status indicates a query error
 			status, ok := statuses[query]
-			failed := (ok && status != fleet.StatusOK)
+			failed := ok && status != fleet.StatusOK
 			err = svc.ingestDistributedQuery(host, query, rows, failed, messages[query])
 		default:
 			err = osqueryError{message: "unknown query prefix: " + query}
 		}
 
 		if err != nil {
-			return osqueryError{message: "failed to ingest result: " + err.Error()}
+			logging.WithExtras(ctx, "ingestion-err", err)
```

Comment:
Should this be `logging.WithErr`?

If not, what do you think of standardizing to make the key for errors be `err`? I think it might be helpful to be more uniform.

Reply:
I suppose we can have a chain of errs instead of just one, yeah. I'll give it a go.

Reply:
What I'm still confused about is whether usage like this should use `logging.WithErr` rather than `logging.WithExtras`?

Reply:
Oh yes, my bad, this would be `WithErr`